bindings/python/crocoddyl/multibody/contacts/contact-1d.cpp

```diff
@@ -103,11 +107,33 @@ void exposeContact1D() {
           bp::make_getter(&ContactData1D::v,
                           bp::return_value_policy<bp::return_by_value>()),
           "spatial velocity of the contact body")
+      .add_property("a0_local",
+                    bp::make_getter(&ContactData1D::a0_local,
+                                    bp::return_internal_reference<>()),
+                    bp::make_setter(&ContactData1D::a0_local),
+                    "desired local contact acceleration")
       .add_property(
-          "a",
-          bp::make_getter(&ContactData1D::a,
-                          bp::return_value_policy<bp::return_by_value>()),
-          "spatial acceleration of the contact body")
+          "dp",
+          bp::make_getter(&ContactData1D::dp,
+                          bp::return_internal_reference<>()),
+          bp::make_setter(&ContactData1D::dp),
+          "Translation error computed for the Baumgarte regularization term")
```

Comment:
```suggestion
          "translation error computed for the Baumgarte regularization term")
```

This is to follow the writing style introduced in Crocoddyl.